src/components/app.js

```diff
@@ -102,6 +102,8 @@ const App = () => {
 
     const [appsAndUrls, setAppsAndUrls] = React.useState([]);
 
+    const [notificationSnackBar, setNotificationSnackBar] = useState();
```

Comment:
Maybe we should use redux store for this as in the future technical errors can come from any part of the application and not only App component?

Reply:
Yes I agree with you, but the goal is to integrate the module and see the display result. Now if we agree to integrate it in all of the applications I have to go through common-ui and make it more generic to display all the errors that can come from any part of the application

Reply:
I am ok with display result and I think we can start working on the component in commons ui.
Just a remark: I guess the message "no merge for this date" was just for testing because we don't need to warn the user for that as it is not an error.

Reply:
Yes I agree with you, I put it here just to testing. Now we will only use it for displaying error.message in catch API calls